app/script/entity/message/Asset.coffee

```diff
@@ -77,7 +77,9 @@ class z.entity.Asset
   @return [Boolean] Is asset of type video
   ###
   is_video: ->
-    if @type is z.assets.AssetType.FILE and @file_type?.startsWith('video') and not z.util.Environment.browser.firefox
+    is_video_asset = @type is z.assets.AssetType.FILE and @file_type?.startsWith 'video'
+    is_unsupported_firefox = z.util.Environment.browser.firefox and platform.version < 49
+    if is_video_asset and not is_unsupported_firefox
```

Comment:
it will be clearer to have the variable named `is_supported_firefox` instead of having double negatives..

so it will become `if is_video_asset and is_supported_firefox`